libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -197,5 +207,6 @@ def _insert_embeddings(
                 session.bulk_save_objects(documents)
                 session.commit()
         except DBAPIError as e:
-            logging.error(e.__cause__)
+            logging.error(f"Insert text failed:\n {e.__cause__}.")
```

Comment:
`Add text failed?` We should verify the error message phrase if they are good enough with PMs